transfer-sh.el

```diff
@@ -132,6 +135,12 @@ If no REMOTE-FILE is given, LOCAL-FILENAME is used."
 
 ;;;###autoload
 (defun transfer-sh-upload (async)
+  "Allias for `tranfer-sh-upload-region'."
```

Comment:
Typo: "Alias".
Use `defalias` instead.

Reply:
Also you might want to mark it as deprecated with `make-obsolete`.

Reply:
Agree, this should use `defalias`, also "Alias"